packages/components/src/draggable/index.js

```diff
@@ -148,6 +149,18 @@ class Draggable extends Component {
 
 	render() {
 		const { children, className } = this.props;
+		if ( typeof children === 'function' ) {
+			return children( {
+				onDraggableStart: this.onDragStart,
+				onDraggableEnd: this.onDragEnd,
+			} );
+		}
+
+		deprecated( 'wp.components.Draggable as a DOM node drag handle', {
```

Comment:
Same here: is there a better way to explain the change?

Reply:
Actually, I'm wondering if we should really deprecate this or just consider it as a fallback in case there's no `children` as function prop provided

Reply:
Unless there is a specific use case where that'd be useful, I'd rather deprecate it. Less code to maintain and understand.